docs/developers/node-tutorial.md

```diff
@@ -607,6 +607,17 @@ We get the following output:
 If you notice from the above output, it returns a `height` of
 `2452` which we will use for the next command.
 
+#### Troubleshooting
+
+If you encounter an error like:
+
+```shell
+$ curl -X POST -d '{"namespace_id": "c14da9d459dc57f5", "data": "4f7a3f1aadd83255b8410fef4860c0cd2eba82e24a", "gas_limit": 60000}'  localhost:26658/submit_pfd
+"rpc error: code = NotFound desc = account celestia1krkle0n547u0znz3unnln8paft2dq4z3rznv86 not found"
+```
+
+It is possible that the account you are trying to submit a PayForData from doesn't have testnet tokens yet. Ensure the testnet faucet has funded your account with tokens and then try again.
```

Comment:
I think line should be limited to 80 chars with Docusaurus, but don't quote me on that.

Reply:
You're right, this repo uses markdownlint [here](https://github.com/rootulp/docs/blob/c3b63d320b119e1522334a8d054218d9cdae4f91/package.json#L15) (no config file) so I would've expected lint CI to fail on this PR.

but [this line](https://github.com/rootulp/docs/blob/c3b63d320b119e1522334a8d054218d9cdae4f91/docs/developers/node-tutorial.md?plain=1#L2) disables the [line length rule](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md013---line-length) in this file presumably because this page contains a lot of code blocks that are easier to read if not broken across multiple lines.

I wrapped the paragraph text in https://github.com/celestiaorg/docs/pull/194/commits/c58076c4e575afb106e9c65df4a809564288dada